src/util/utils.js

```diff
@@ -23,8 +23,13 @@ const responseStatusHandler = (status, entity, id, url) => {
   }
 };
 
+const getIntegrationVersion = () => {
```

Comment:
It doesn't have to be a function, just a constant.

Reply:
Added for future extension